### PR TITLE
RFC: auto-fixes from ruff 0.6 (table)

### PR DESCRIPTION
**Pull request by neutrinoceros**
### Description
Partially address #16863 for `table`

- [ ] By checking this box, the PR author has requested that maintainers do **NOT** use the "Squash and Merge" button. Maintainers should respect this when possible; however, the final decision is at the discretion of the maintainer that merges the PR.

### CODE REVIEW

File: astropy/table/pprint.py [hunk 1]
```diff
@@ -841,8 +839,6 @@ def _more_tabcol(
        h : print this help""",
                     end=" ",
                 )
-            if i0 < 0:
-                i0 = 0
-            if i0 >= len(tabcol) - delta_lines:
-                i0 = len(tabcol) - delta_lines
+            i0 = max(i0, 0)
+            i0 = min(len(tabcol) - delta_lines, i0)
```
Comment (pllim) [inline astropy/table/pprint.py:843]: Huh. Can these two be combined?

File: astropy/table/pprint.py [hunk 2]
```diff
@@ -213,15 +213,13 @@ def _get_pprint_size(max_lines=None, max_width=None):
...
         elif max_lines < 0:
             max_lines = sys.maxsize
-        if max_lines < 8:
-            max_lines = 8
+        max_lines = max(max_lines, 8)
 
         if max_width is None:
             max_width = width
         elif max_width < 0:
             max_width = sys.maxsize
```
Comment (pllim) [inline astropy/table/pprint.py:221]: What about this?

Comment (neutrinoceros) [reply astropy/table/pprint.py:221]: How should it be refactored ? I don't see it.

Comment (pllim) [reply astropy/table/pprint.py:221]: Ah, right. I guess the comparisons are different enough we can't turn it into a one-liner. Never mind. Sorry for the noise.

**Review by mhvk — APPROVED** [review]
Clear improvements, I would say!

File: astropy/table/pprint.py [hunk 3]
```diff
@@ -213,15 +213,13 @@ def _get_pprint_size(max_lines=None, max_width=None):
             max_lines = lines
         elif max_lines < 0:
             max_lines = sys.maxsize
-        if max_lines < 8:
-            max_lines = 8
+        max_lines = max(max_lines, 8)
```
Comment (mhvk) [inline astropy/table/pprint.py:216]: On a second look after finding it was slower, I'm less sure about this case: it follows a whole slew of other size comparisons using `if`, so perhaps nicer to keep it as is?

Comment (pllim) [reply astropy/table/pprint.py:216]: Hmm... I applied `benchmark` label FWIW.

Comment (taldcroft) [reply astropy/table/pprint.py:216]: Please revert to the original for performance and readability. Note this continues the pattern of checks from above.

File: astropy/table/pprint.py [hunk 4]
```diff
@@ -841,8 +839,5 @@ def _more_tabcol(
        h : print this help""",
                     end=" ",
                 )
-            if i0 < 0:
-                i0 = 0
```
Comment (taldcroft) [inline astropy/table/pprint.py:845]: Please revert to the original for these two lines.

**Comment by neutrinoceros** [comment]
If I read correctly, @taldcroft's review can be summarized as "revert everything".
Noqa comments are not needed either (following https://github.com/astropy/astropy/issues/16863#issuecomment-2304713935) so let's just close this one.

**Pull request closed without merging**